robustus/robustus.py

```diff
@@ -553,7 +553,7 @@ def install(self, args):
                 active_branch = re.search('origin/\w*', msg).group(0)
                 # http://stackoverflow.com/questions/6657690/python-getoutput-equivalent-in-subprocess
             except Exception as err:
-                active_branch = '<Could not find active branch - %s: %s>' % (err.__class__.__name__, err.message)
+                active_branch = '<Could not find active branch - %s: %s>. Branch out of date. Try running git pull' % (err.__class__.__name__, err.message)
```

Comment:
I'm getting the same error when I switch branches (ie to ppm_utils) and run source use_repo.sh. I tried git pull origin ppm_utils as well as git pull origin master while in the ppm_utils branch, but that doesn't seem to solve the problem. I'll look into it a bit more and make the error message a little more explicit. @bc-jmoork, fyi.